lib/bcchelper.py

```diff
@@ -399,7 +399,7 @@ def key_type_string(self, key, key_type):
           DEFAULT_KEY_TYPE - no post-processing needed
           IP_KEY_TYPE - strips unwanted symbols from IPV4 literal address
         """
-        keystr = str(getattr(key, key_type[0]))
+        keystr = getattr(key, key_type[0]).decode('utf-8')
```

Comment:
Since the `error` default argument is `strict` for `decode()` - it could raise an exception
Instead would be better to pass either `replace` or `backslashreplace` ?